focuspoints.lrdevplugin/FocusPoint.lua

```diff
@@ -14,27 +14,37 @@
   limitations under the License.
 --]]
 
-local LrSystemInfo = import 'LrSystemInfo'
 local LrFunctionContext = import 'LrFunctionContext'
 local LrApplication = import 'LrApplication'
 local LrDialogs = import 'LrDialogs'
-local LrView = import 'LrView'
```

Comment:
Which delegates does it break? 
I tried it on a variety of test images I have and could not find any issues.